docs/sql/system.txt

```diff
@@ -688,6 +688,57 @@ _node
+| Column Name                              | Description                                        | Return Type |
++==========================================+====================================================+=============+
+| ``recovery``                             | Represents recovery statistic of the particular    | ``Object``  |
+|                                          | shard.                                             |             |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['stage']``                    | Recovery stage:                                    | ``String``  |
+|                                          |                                                    |             |
+|                                          | * init: Recovery has not started                   |             |
+|                                          | * index: Reading index meta-data and copying bytes |             |
+|                                          |   from source to destination                       |             |
+|                                          | * start: Starting the engine;                      |             |
+|                                          |   opening the index for use                        |             |
+|                                          | * translog: Replaying transaction log              |             |
+|                                          | * finalize: Cleanup                                |             |
+|                                          | * done: Complete                                   |             |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['type']``                     | Recovery type:                                     | ``String``  |
+|                                          |                                                    |             |
+|                                          | * store                                            |             |
+|                                          | * snapshot                                         |             |
+|                                          | * replica                                          |             |
+|                                          | * relocating                                       |             |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['size']``                     | Shards recovery statistic in bytes.                | ``Object``  |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['size']['total_bytes']``      | Total number of bytes in the shard.                | ``Long``    |
```

Comment:
would it make sense to remove the `_bytes` suffixes? In the fs column of sys.nodes we also have just size/used